src/dosext/mfs/mfs.c

```diff
@@ -381,17 +381,6 @@ void mfs_set_stk_offs(int offs)
   stk_offs = offs;
 }
 
-static int cds_drive(cds_t cds)
```

Comment:
Maybe it would be good to leave `cds_drive()`
and just make it to use first letter? Fewer changes,
plus someone in the future may change it again. :)